Export SetResourceConstraints, otherwise chromium won't build; bug was reported
upstream already: http://code.google.com/p/v8/issues/detail?id=506

--- a/include/v8.h
+++ b/include/v8.h
@@ -2122,7 +2122,7 @@
 };
 
 
-bool SetResourceConstraints(ResourceConstraints* constraints);
+bool V8EXPORT SetResourceConstraints(ResourceConstraints* constraints);
 
 
 // --- E x c e p t i o n s ---
